visualize-admin/visualization-tool
# Hierarchy published

Now in published view, the hierarchy is visible (same style as in the left filters = flat hierachy).



We should show the hierarchy as a tree in a subsequent PR.

## How to test

Since we cannot publish on Vercel previews, I think it's better to test locally.
After publishing a chart with interactive filters on a dimension having hierarchie, values in the select should be grouped.
You can use "bathing water hierarchie" (draft), and use the "Observation location" dimension as interactive filters.

Fix https://github.com/visualize-admin/visualization-tool/issues/739
